includes/namespace.php

```diff
@@ -129,10 +130,8 @@ function remove_site( $error, $site ) {
  * @return void
  */
 function deactivate( $network_wide ) {
-	unregister_post_type( Story_Post_Type::POST_TYPE_SLUG );
-	if ( ! defined( '\WPCOM_IS_VIP_ENV' ) || false === \WPCOM_IS_VIP_ENV ) {
-		flush_rewrite_rules( false ); // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.flush_rewrite_rules_flush_rewrite_rules
-	}
```

Comment:
This code was removed without replacement.

It ensured that rewrite rules are properly flushed upon plugin deactivation.

Related: why aren't we doing that on plugin _activation_ as well? 🤔

Reply:
This happens here now. 

https://github.com/google/web-stories-wp/pull/6906/files#diff-afc0298087d72ee3c18032663a02b2c9f484a7bd1262fe8956c0300b8fb0305cR141-R143

It does now do that on deactivation. 

https://github.com/google/web-stories-wp/pull/6906/files#diff-afc0298087d72ee3c18032663a02b2c9f484a7bd1262fe8956c0300b8fb0305cR163-R165

Reply:
Sorry, these links don't seem to work for me. But I found it now in `ServiceBasedPlugin`